src/utilities/new_versions.jl

```diff
@@ -219,7 +227,8 @@ function make_pr_for_new_version(
     brand_new_compat = new_compat_entry(
         entry_type, dep.version_verbatim, compat_entry_for_latest_version
     )
-    new_pr_title, new_pr_body = pr_info(
+    new_pr_title,
+    new_pr_body = pr_info(
```

Comment:
Keep on same line?

Reply:
I'd like to, but the formatter keeps moving it :/

Reply:
oh hmm nevermind, the formatter says it should be on the same line now